bundle/builder.go

```diff
@@ -0,0 +1,67 @@
+package bundle
+
+import (
+	"bytes"
+	"context"
+	"io/fs"
+	"testing/fstest"
+
+	layerfs "github.com/dschmidt/go-layerfs"
+	opaBundle "github.com/open-policy-agent/opa/bundle"
+	opaCompile "github.com/open-policy-agent/opa/compile"
+)
+
+func Build(fs fs.FS) ([]byte, error) {
+
+	buf := bytes.NewBuffer(nil)
+
+	b, err := createBundle(fs)
+	if err != nil {
+		return nil, err
+	}
+
+	compiler := opaCompile.New().
+		WithBundle(&b).
+		WithOutput(buf)
+
+	err = compiler.Build(context.Background())
+	if err != nil {
+		return nil, err
+	}
+
+	return buf.Bytes(), nil
+	
+}
+
+func createBundle(fs fs.FS) (opaBundle.Bundle, error) {
+	loader, err := opaBundle.NewFSLoader(fs)
+	if err != nil {
+		return opaBundle.Bundle{}, err
+	}
+
+	reader := opaBundle.NewCustomReader(loader)
+
+	b, err := reader.Read()
+	if err != nil {
+		return opaBundle.Bundle{}, err
+	}
+
+	return b, nil
+}
+
+func CISKubernetesFS() fs.FS {
+	return layerfs.New(CommonEmbed, CISRulesEmbed)
+}
+
+func CISEksFS() fs.FS {
+	return layerfs.New(CommonEmbed, EKSRulesEmbed)
+}
+
+func MergedFSWithDataYaml(fs fs.FS, dataYaml string) fs.FS {
+	yamlFS := fstest.MapFS{
```

Comment:
Can we avoid using test libraries?
Or is this considered the best practice in this case?

Reply:
I'm using this library to create an FS out of a single data.yaml file that includes content.
I can create this abstraction by myself, but I assume that for this tiny scenario the library should handle it without any problems.
I'm opening a gap regarding this building flow and will try to advance ourselves to pre-build bundles instead of building them on run-time. So hopefully this code will be removed in the near future.
Due to that, I think that we can use this library for this case.
What do you think? @eyalkraft